<launch>
  <node pkg="motion_capture_system"
    type="mocap_vicon_node"
    name="vicon_new"
    output="screen">
    <param name="vicon_server" value="alkaline2"/>
    <param name="frame_rate" value="100"/>
    <param name="max_accel" value="10.0"/>
    <param name="publish_tf" value="false"/>
    <param name="fixed_frame_id" value="mocap"/>
    <rosparam param="model_list">[]</rosparam>
  </node>
</launch>
